subsys/suit/stream/Kconfig

```diff
@@ -44,14 +44,14 @@ if SUIT_STREAM_SINK_EXTMEM
 
 config SUIT_STREAM_SINK_EXTMEM_ADDRESS_RANGE_START
 	hex
-	default 0x60000000 if SOC_NRF54H20
+	default 0x60000000 if SOC_SERIES_NRF54HX
 	help
 	  Start address of the extended memory range.
 	  This value is SOC specific and is not meant to be changed.
 
 config SUIT_STREAM_SINK_EXTMEM_ADDRESS_RANGE_SIZE
 	hex
-	default 0x40000000 if SOC_NRF54H20
+	default 0x40000000 if SOC_SERIES_NRF54HX
 	help
 	  Size of the extended memory range.
 	  This value is SOC specific and is not meant to be changed.
```

Comment:
I don't get why this is here, you already have a Kconfig (which needs to move to dts) but why is the option duplicated already?

Reply:
Nice catch!
This is a leftover and can be safely removed.